### ISSUE TITLE
I can not find the noteAppBehaivor in the demo but yes in the other repo

### ISSUE THREAD

**Issue by Borjagodoy** [firebase/polymerfire#68]
### Description
I can not find the noteAppBehaivor in the demo but yes in the other repo, the file i have found in https://github.com/PolymerLabs/note-app/blob/master/common/note-app/na-behavior.html  if they can be helpful.

Thanks

**Comment by dmix007** [comment]
I agree.  Adding the demo the the polymerfire repo would be very helpful. I also noticed the demo is not available for viewing at https://elements.polymer-project.org/elements/polymerfire.
